### PR TITLE
S01e02 schemes update

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/icraf_s01e02.json [hunk 1]
```diff
@@ -3,69 +3,373 @@
...
+        "CodeType": "Normal",
+        "DisplayText": "eurura ecocide warriors shg",
+        "NumericValue": 10,
+        "StringValue": "eurura_ecocide_warriors_shg",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "4b8ecb93",
+        "CodeType": "Normal",
+        "DisplayText": "catholic_diocese_murang'a",
```
Comment (as2388) [inline code_schemes/icraf_s01e02.json]: The DisplayText and StringValue seem to be swapped here

File: code_schemes/icraf_s01e02.json [hunk 2]
```diff
@@ -3,69 +3,373 @@
...
+        "NumericValue": 29,
+        "StringValue": "unesco",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-c28fa84a",
+        "CodeType": "Normal",
+        "DisplayText": "ldk(K)ltd athi river",
+        "NumericValue": 30,
+        "StringValue": "ldk(K)ltd_athi_river",
```
Comment (as2388) [inline code_schemes/icraf_s01e02.json]: maybe substitute the brackets for _? Not sure

File: code_schemes/icraf_s01e02.json [hunk 3]
```diff
@@ -3,69 +3,373 @@
...
+        "NumericValue": 34,
+        "StringValue": "asking_for_specific_support",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-57c1e449",
+        "CodeType": "Normal",
+        "DisplayText": "complains absence of gov/extension workers",
+        "NumericValue": 35,
+        "StringValue": "complains_absence_of_gov/extension_workers",
```
Comment (as2388) [inline code_schemes/icraf_s01e02.json]:
```suggestion
        "StringValue": "complains_absence_of_gov_extension_workers",
```

File: code_schemes/icraf_s01e02.json [hunk 4]
```diff
@@ -3,69 +3,373 @@
...
         "DisplayText": "NIC (not internally consistent)",
         "NumericValue": -50,
         "StringValue": "NIC",
         "VisibleInCoda": false
-      },
-      {
+        },
+        {
+        "StringValue": "Noise_Other_Project",
+        "ControlCode": "NOP",
```
Comment (as2388) [inline code_schemes/icraf_s01e02.json:349]: I presume this came out of discussion with @lukechurch. Does this code need to be added to Core?

**Review by as2388 — APPROVED** [review]
PS I don't think this PR needed to depend on #14. In which case, if you had branched from master instead of generate-advert-phone-numbers you would have been able to merge this without waiting for 14.

File: code_schemes/icraf_s01e02.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e02.json]: Updated thanks

File: code_schemes/icraf_s01e02.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e02.json]: Updated thanks

File: code_schemes/icraf_s01e02.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e02.json]: Updated thanks

File: code_schemes/icraf_s01e02.json (hunk 4 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e02.json:349]: Yes. Maybe we can add it to the core @lukechurch ? Its however project specific at the moment.

**Pull request merged**